stable/coredns/templates/podsecuritypolicy.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.pspEnable }}
-apiVersion: extensions/v1beta1
+apiVersion: policy/v1beta1
```

Comment:
by using capabilities: https://helm.sh/docs/topics/chart_template_guide/builtin_objects/ the deployment can work on newer and older clusters.

Reply:
Cool, I didn't know about capabilities, I just added them.